src/aat/resources/features/F-200 - New Access Control Create Case/S-200.4.td.json

```diff
@@ -0,0 +1,78 @@
+{
+  "_guid_": "S-200.4",
+
+  "productName": "CCD Data Store",
+  "operationName": "create case",
+  "method": "POST",
+  "uri": "/case-types/{ctid}/cases",
+
+  "specs": [
+    "a role assignment to access CaseType CT2",
+    "attempts to create a case for CT2"
+  ],
+
+  "users" : {
+    "invokingUser": {
+      "username": "ccd.ac.solicitor1@gmail.com",
+      "password": "Pa55word11"
```

Comment:
lets hide the password with an env var, can just use [[$CCD_BEFTA_CASEWORKER_1_PWD]] for now until we create new env vars in vault and jenkins file